FullStoryMiddlewareExample/main.m

```diff
@@ -0,0 +1,15 @@
+//
+//  main.m
+//  FullStoryMiddlewareExample
+//
+//  Created by FullStory on 9/24/20.
+//  Copyright (c) 2020 FullStory. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "AppDelegate.h"
+
+int main(int argc, char * argv[]) {
+    NSString * appDelegateClassName;
```

Comment:
You removed the autorelease pool, but you also removed getting the name of the app delegate's class.
```suggestion
    NSString * appDelegateClassName = NSStringFromClass([AppDelegate class]);
```